svc/FetchService.js

```diff
@@ -37,10 +52,12 @@ export class FetchService {
      *      with the request body (for POSTs/PUTs sending form-url-encoded).
      * @param {string} [opts.method] - HTTP Request method to use for the request. If not specified,
      *      the method will be set to POST if there are params, otherwise GET.
-     * @param {string} [opts.contentType] - value to use in the Content-Type header in the request.
-     *     If not specified, contentType is set based on method: 'application/x-www-form-urlencoded'
-     *     for POSTs, 'text/plain' otherwise.
-     * @param {boolean} [opts.acceptJson] - true to set Accept header to 'application/json'.
+     * @param {(Object|Headers)} [opts.headers] - headers to send with this request. If an Object,
```

Comment:
Out of curiosity, why do we take an actual Headers object?  Do we have a use case for that right now?  Seems like we could leave it out for now, leaving us more flexibility to add it with different semantics in the future when we need it.

Reply:
It's there to offer a way to take full control of the headers for that particular request - i.e. bypass any default headers the app is providing. We have a use-case with export right now where we need to make sure we *don't* set a Content-Type header:
https://github.com/exhi/hoist-react/blob/develop/svc/GridExportService.js#L81